core/src/empty/empty.tsx

```diff
@@ -1,30 +1,18 @@
 import s from "./empty.module.css";
-import { createElement } from "react";
-import { Button } from "../button/button";
-import cloudImg from "./empty1.jpg";
+import cloudSVG from "./empty1.svg";
 
 interface EmptyProps {
-    message: string,
-    actionLabel: string,
-    actionHandler?: React.MouseEventHandler
+	message: string;
+	action?: React.ReactNode;
 }
 
 export const Empty = (props: EmptyProps): JSX.Element => {
-    const { message, actionLabel, actionHandler } = props;
-    return (
-        <div className={s.container}>
-            <img src={cloudImg} />
-            <p className={s.errorText}>{message}</p>
-            {
-                actionHandler && <Button
-                    onClick={actionHandler}
-                    highlight
-                    size={Button.sizes.medium}
-                    style={Button.styles.outset}
-                >
-                    {actionLabel}
-                </Button>
-            }
-        </div>
-    );
+	const { message, action } = props;
+	return (
+		<div className={s.container}>
+			<img src={cloudSVG} />
```

Comment:
The img tag here is missing too many attributes: "alt", "width", "height". Please provide them